bus-mapping/src/evm/opcodes/sload.rs

```diff
@@ -71,9 +71,9 @@ impl Opcode for Sload {
         // Storage read
         let value = geth_step.storage.get_or_err(&key)?;
 
-        let is_warm = state
+        let is_warm_prev = !state
             .sdb
-            .check_account_storage_in_access_list(&(contract_addr, key));
+            .add_account_storage_to_access_list((contract_addr, key));
```

Comment:
Note: as in previous comments, the statedb write should already be handled by `push_op_reversible`